plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -1196,6 +1222,10 @@ private void getPifs() {
         s_logger.debug("done looking for pifs, no more bridges");
     }
 
+    boolean isGuestBridge(String bridge) {
```

Comment:
Thanks! Removing the duplicate code is great 👍
If you want, you can go a step further creating a test case for `isGuestBridge(String bridge)` and `isPublicBridge(String bridge)` ;)